README.md

```diff
@@ -9,39 +11,93 @@ Sesh is a session management library written in Go. It uses a postgres table to
 
 ## Configuration
 
-1. Run the migration included in ./migrations to set up the `sessions` table in your postgres db.
-2. Instantiate the sesh.Sessions struct
+1. Create and configure an scs.SessionManager
+2. Implement Sesh Interfaces
+3. Pass the UserSessionManager anywhere it is needed
+
+For a complete example of configuring sesh to protect routes, see /pkg/dummy/dummy.go
+
+### Create and configure an scs.SessionManager
+
+Sesh uses SCS to store session data backed by a session cookie. In order for Sesh to work you must configure SCS first. From the scs README:
 
 ```
-    dbConnection, err := sqlx.Open("postgres", dbConnectionURL)
-	if err != nil {
-		return nil, fmt.Errorf("error connecting to database using sqlx.Open: %w", err)
-	}
+	// Initialize a new session manager and configure the session lifetime.
+	sessionManager = scs.New()
+	sessionManager.Lifetime = 24 * time.Hour
 
-    seshLogger := AuthLogger{}
-	sessions := sesh.NewSessions(dbConnection, seshLogger, 5*time.Minute, false)
+	mux := http.NewServeMux()
+	mux.HandleFunc("/put", putHandler)
+	mux.HandleFunc("/get", getHandler)
+
+	// Wrap your handlers with the LoadAndSave() middleware.
+	http.ListenAndServe(":4000", sessionManager.LoadAndSave(mux))
+}
 ```
 
-There are several options, described below.
+The LoadAndSave() middleware must wrap _all_ your handlers, it provides arbitrary session data storage. Sesh uses that data store to store information about authenticated user sessions.
+
+### Implement Sesh Interfaces
+
+Sesh limits users to a single concurrent session. If a user logs in while they have an active session, that session will be invalidated and all future requests from that session will be rejected in favor of the new session. In order to do that we need to keep track of the current session identifier for every signed in user. Rather than maintain its own table to track this information, necessitating an additional db hit on every request, sesh expects that info to be stored directly on your user table and is accessed via the following interfaces:
+
```

Comment:
Would it be helpful to show the golang interfaces here?  They are small and it might help to give a concrete snippet